test/test_less_pycharm.py

```diff
@@ -0,0 +1,391 @@
+)
+beam2 = cheetah.ParticleBeam.from_astra(
+    "D:/Fachpraktikum_DESY/GitHub/cheetah/benchmark/cheetah/ACHIP_EA1_2021.1351.001"
+)
+
+
+def test_import_ParameterBeam():
+    assert (
+        str(beam1)
+        == "ParameterBeam(mu_x=0.000001, mu_xp=0.000000, mu_y=-0.000002, mu_yp=-0.000000, sigma_x=0.000175, sigma_xp=0.000004, sigma_y=0.000175, sigma_yp=0.000004, sigma_s=0.000008, sigma_p=0.002280, energy=107315902.444)"
+    )
+
+
+def test_import_ParticleBeam():
+    assert (
+        str(beam2)
+        == "ParticleBeam(n=100000, mu_x=0.000001, mu_xp=0.000000, mu_y=-0.000002, mu_yp=-0.000000, sigma_x=0.000175, sigma_xp=0.000004, sigma_y=0.000175, sigma_yp=0.000004, sigma_s=0.000008, sigma_p=0.002280, energy=107315902.444)"
+    )
+
+
+def test_beam1_mu():
+    actual = beam1._mu
+    expected = torch.tensor(
+        [
+            8.2413e-07,
+            5.9885e-08,
+            -1.7276e-06,
+            -1.1746e-07,
+            5.7250e-06,
+            3.8292e-04,
+            1.0000e00,
+        ],
+        dtype=torch.float32,
+    )
+    assert torch.allclose(actual, expected, rtol=1e-5, atol=1e-8, equal_nan=False)
+
+
+def test_beam2_particles_mean():
+    actual = beam2.particles.mean(axis=0)
+    expected = torch.tensor(
+        [
+            8.2413e-07,
+            5.9885e-08,
+            -1.7276e-06,
+            -1.1746e-07,
+            5.7250e-06,
+            3.8292e-04,
+            1.0000e00,
+        ],
+        dtype=torch.float32,
+    )
+    assert torch.allclose(actual, expected, rtol=1e-5, atol=1e-8, equal_nan=False)
+
+
+def test_division_beam1_beam2():
+    actual = beam1._mu / beam2.particles.mean(axis=0)
+    expected = torch.tensor([1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000])
+    assert torch.allclose(actual, expected, rtol=1e-5, atol=1e-8, equal_nan=False)
+
+
+def test_division_beam2_beam1():
+    actual = beam2.particles.mean(axis=0) / beam1._mu
+    expected = torch.tensor([1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000])
+    assert torch.allclose(actual, expected, rtol=1e-5, atol=1e-8, equal_nan=False)
+
+
+def test_division_cov_beam1_beam2():
+    actual = beam1._cov / np.cov(beam2.particles.t().numpy())
+    expected = torch.tensor(
+        [
+            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, float("nan")],
+            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, float("nan")],
+            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, float("nan")],
+            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, float("nan")],
+            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, float("nan")],
+            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, float("nan")],
+            [
+                float("nan"),
+                float("nan"),
+                float("nan"),
+                float("nan"),
+                float("nan"),
+                float("nan"),
+                float("nan"),
```

Comment:
In order to make it possible to add numbers to mu you introduced a 7th variable. Since the mu vector now has 7 states, the transfer function needs to be adapted to, such that the matrix multiplication is still possible and the dimension of the output vector is also 7. This means the transfer map must be a 7x7 matrix. Since the transfer map is a 7x7 matrix the covariance matrix, which is also multiplied with the transfer map, has to be of Dimension 7x7. The covariance matrix with pysical meaning is of dimension 6x6. 
Now an additional column as well as an additional row of 0 are introduced to the matrix to ensure dimension 7x7.
Because of this, there happens to be a nan in all of the places where there is a zero when calculating the division of 2 covariance matrices, as 0/0 is not defined.

Reply:
As we have discussed I replaced the division by a comparison of the `ParameterBeam` and the `ParticleBeam`.